.github/workflows/deploy-docs.yml

```diff
@@ -25,6 +25,6 @@ jobs:
         run: yarn build-storybook
 
       - name: Deploy Storybook
-        run: yarn deploy-storybook -- --ci --existing-output-dir=storybook-static
+        run: npm run deploy-storybook -- --ci --existing-output-dir=storybook-static
```

Comment:
Yarn don't fully support all flags here, maybe this is the problem when deploying to prod